src/play.c

```diff
@@ -242,7 +255,7 @@ static int aufile_load(struct mbuf *mb, const char *filename,
 	mem_deref(af);
 
 	if (!err) {
-		mb->pos = 0;
+		mb->pos = (prm.srate * fsize * prm.channels / 1000) / file_size;
```

Comment:
I think you can use `aufmt_sample_size(prm.fmt)` instead `fsize` here.

Reply:
Yes, that's much short and more complete (done)